content/en/docs/pipelines/sdk/python-function-components.ipynb

```diff
@@ -60,7 +61,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "3. Create an instance of the [`kfp.Client` class][kfp-client].\n",
+    "3. Create an instance of the [`kfp.Client` class][kfp-client]. To find your\n",
+    "   Kubeflow Pipelines cluster's host name, open the Kubeflow Pipelines user\n",
```

Comment:
Small observation: do we use `host name` of `hostname`? Source: https://cloud.google.com/s/results?q=host%20name

Reply:
Good point, that should be hostname.

Reply:
Done